components/ws-manager-api/core.proto

```diff
@@ -700,3 +703,17 @@ message WorkspaceClass {
     string Id = 1;
     string DisplayName = 2;
 }
+
+message RestartRing1Request {
```

Comment:
I'd expect this to be routed through the supervisor/workspacekit connection that already exists. A much shorter path compared to workspace -> server -> ws-manager -> ws-daemon -> IWS -> workspacekit.